docs/04-operation-guides/troubleshooting/obsv-03-troubleshoot-loki-logging.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Loki doesn't show the logs you want to see
+---
+
+## Condition
+
+Loki shows fewer logs than you would like to see.
+
+## Cause
+
+By default, Loki stores up to 30 GB of data for a maximum of 5 days, with maximum ingestion rate of 3 MB/s. If the default size or time is exceeded, the oldest logs are removed first.
```

Comment:
Do you really want to repeat that values 3 times?

Reply:
I absolutely don't! That's why the PR's still on hold, I wasn't finished thinking :D

It would be best to keep it in the Opsguide and remove it from the other two docs. What stops me from removing it from the Overview doc is the fact that I'm not sure where to put the Prometheus limits then (and the information that traces aren't stored persistently and are gone after a restart).

Reply:
removed it from the instructional docs, only keeping it in the Overview